java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -168,18 +169,17 @@ public static TimestampedEntry parseTimestampedEntry(InputStream in) {
       int length = (int) readNumber(in, 3);
       signedEntry.x509 = readFixedLength(in, length);
     } else if (entryType == Ct.LogEntryType.PRECERT_ENTRY_VALUE) {
-      Ct.PreCert.Builder preCertBuilder = Ct.PreCert.newBuilder();
+      PreCert preCert = new PreCert();
 
-      byte[] arr = readFixedLength(in, 32);
-      preCertBuilder.setIssuerKeyHash(ByteString.copyFrom(arr));
+      preCert.issuerKeyHash = readFixedLength(in, 32);
 
       // set tbs certificate
-      arr = readFixedLength(in, 2);
+      byte[] arr = readFixedLength(in, 2);
```

Comment:
On second thought (and a bit of discussion here), I'll leave this PR to be neutral in terms of what it does (stick to what's in the description), rather than bury a possible bugfix in it. I'll send a separate PR for that.